<ui version="4.0" stdsetdef="1" >
  <author>Frerich Raabe &lt;raabe@kde.org></author>
  <class>TagGuesserConfigDlgWidget</class>
  <widget class="QWidget" name="TagGuesserConfigDlgWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>480</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item rowspan="5" row="0" column="0" colspan="1" >
        <widget class="K3ListView" name="lvSchemes" >
          <property name="frameShape" >
            <enum>StyledPanel</enum>
          </property>
          <property name="frameShadow" >
            <enum>Sunken</enum>
          </property>
          <property name="fullWidth" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Currently used file name schemes</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Here you can see the currently configured file name schemes which the "Suggest" button in the tag editor uses to extract tag information from a file name. Each string may contain one of the following placeholders:&lt;ul>
&lt;li>%t: Title&lt;/li>
&lt;li>%a: Artist&lt;/li>
&lt;li>%A: Album&lt;/li>
&lt;li>%T: Track&lt;/li>
&lt;li>%c: Comment&lt;/li>
&lt;/ul>
For example, the file name scheme "[%T] %a - %t" would match "[01] Deep Purple - Smoke on the water" but not "(Deep Purple) Smoke on the water". For that second name, you would use the scheme "(%a) %t".&lt;p/>
Note that the order in which the schemes appear in the list is relevant, since the tag guesser will go through the list from the top to the bottom, and use the first matching scheme.</string>
          </property>
          <column>
            <property name="text" >
              <string>File Name Scheme</string>
            </property>
            <property name="clickable" >
              <bool>false</bool>
            </property>
            <property name="resizable" >
              <bool>false</bool>
            </property>
          </column>
        </widget>
      </item>
      <item rowspan="1" row="1" column="1" colspan="3" >
        <widget class="KPushButton" name="bAdd" >
          <property name="text" >
            <string>&amp;Add</string>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Add a new scheme</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Press this button to add a new file name scheme to the end of the list.</string>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="QToolButton" name="bMoveUp" >
          <property name="text" >
            <string/>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Move scheme up</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Press this button to move the currently selected scheme one step upwards.</string>
          </property>
        </widget>
      </item>
      <item row="0" column="3" >
        <widget class="QToolButton" name="bMoveDown" >
          <property name="text" >
            <string/>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Move scheme down</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Press this button to move the currently selected scheme one step downwards.</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="2" column="1" colspan="3" >
        <widget class="KPushButton" name="bModify" >
          <property name="text" >
            <string>&amp;Modify</string>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Modify scheme</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Press this button to modify the currently selected scheme.</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="3" column="1" colspan="3" >
        <widget class="KPushButton" name="bRemove" >
          <property name="text" >
            <string>&amp;Remove</string>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Remove scheme</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>Press this button to remove the currently selected scheme from the list.</string>
          </property>
        </widget>
      </item>
      <item row="4" column="2" >
        <spacer name="spacer1" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>130</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
    <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header>k3listview.h</header>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >k3listview.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
</ui>
